iocage/lib/Config/Resource/ResourceConfig.py

```diff
@@ -28,11 +28,11 @@
 
 class ResourceConfig(iocage.lib.Config.Prototype.Prototype):
 
-    resource: 'iocage.lib.Resource.Resource' = None  # type: ignore
+    resource: 'iocage.lib.Resource.Resource' = None
 
     def __init__(
         self,
-        resource: 'iocage.lib.Resource.Resource',  # type: ignore
+        resource: 'iocage.lib.Resource.Resource',
```

Comment:
why is this a forward declaration?

Reply:
Resource includes ResourceConfig, so that ResourceConfig cannot reference Resource with a non-string literal.